esp/bindings/http/platform/httptransport.cpp

```diff
@@ -1565,11 +1565,10 @@ void CHttpRequest::parseCookieHeader(char* cookiestr)
         if(name.charAt(0) != '$')
         {
             cookie = new CEspCookie(name.str(), value.str());
+            cookie->setPath("/");
             cookie->setVersion(version);
             m_cookies.append(*cookie);
         }
-        else if(stricmp(name.str(), "$Path") == 0 && cookie != NULL)
-            cookie->setPath(value.str());
```

Comment:
Not sure whether it is safe or not, The existing code uses the Path in the appendToRequestHeader() and appendToResponseHeader(). What if a third party ESP client needs it?